cmd/test/subdomain-release_test.go

```diff
@@ -17,13 +17,13 @@ func TestSubdomainRelease(t *testing.T) {
 	p.Run()
 	require.Equal(t, true, equalsEmptySubdomainList(p.Stdout()))
 	//Make sure subdomain was reserved
-	p = testcli.Command(exePath, "subdomain", "reserve", "testdomain", "--config", configPath)
+	p = testcli.Command(exePath, "subdomain", "reserve", "-s", "testdomain", "--config", configPath)
```

Comment:
I thought we removed the "subdomain" subcommand?

Reply:
Yes these need to be all redone but the testcli was broken for me so i couldnt fix it yet. Need to change to another cli test tool